sw/cheri/common/flash-utils.hh

```diff
@@ -46,10 +46,10 @@ class SpiFlash
 	{
 		set_cs(true);
 		spi->blocking_write(&CmdEnableReset, 1);
-		set_cs(false);
```

Comment:
The winbond datasheet says that instructions should be initiated with the falling edge of C̅S̅ and completed with rising edge of it.